README.md

```diff
@@ -64,6 +64,25 @@ Flavors can be built in parallel easily with the `build.parallel` target:
 make FLAVORS="<flavor> <other flavor> <...>" build.parallel
 ```
 
+### Building with a specific version of Ceph
+Some distributions can select a specific version of Ceph to install.
+You just have to pad the required version to the ceph release code name.
+
+The required version will be saved in ``CEPH_POINT_RELEASE`` variable (including the version separator).
+The version separator is usually a dash ('-') or an equal sign ('=').
+``CEPH_POINT_RELEASE`` remains empty if no point release is given.
+
+Note that ``CEPH_VERSION`` variable still feature the ceph code name, **luminous** in this example.
+
+```
```

Comment:
Having output from make in the doc is a bad idea. It's not strictly necessary, and the make output may change over time, and then this easily becomes outdated.

Reply:
ok